.mailmap

```diff
@@ -0,0 +1,4 @@
+Jason Dobry <jason.dobry@gmail.com> Jason Dobry <jmdobry@users.noreply.github.com>
```

Comment:
This file looks pretty not great.

Reply:
The purpose of `.mailmap` is to remove duplicates from the generated `CONTRIBUTORS` file. `.mailmap` files would need to be created manually in each repo as necessary. Or we could just not have them and live with multiple entries with different emails for a single individual in all the `CONTRIBUTORS` files.

Reply:
I see.